client/lib/directly/index.js

```diff
@@ -1,4 +1,10 @@
 /**
+ * @file Interface to the Directly API. See README.md for more information about Directly and a few gotchas.
+ *
+ * @see https://cloudup.com/cySVQ9R_O6S
```

Comment:
note that we can add an explanation with `@see` - _why_ should I see that reference?